src/assisted_installer_controller/operator_handler.go

```diff
@@ -62,7 +76,7 @@ func (c *controller) isOperatorAvailable(handler OperatorHandler) bool {
 }
 
 func (c *controller) isOperatorAvailableInService(operatorName string, openshiftVersion string) (*models.MonitoredOperator, bool) {
-	operatorStatusInService, err := c.ic.GetClusterMonitoredOperator(utils.GenerateRequestContext(), c.ClusterID, operatorName, openshiftVersion)
+	operatorStatusInService, err := c.ic.GetClusterMonitoredOperator(utils.GenerateRequestContext(), c.ClusterID, operatorName, openshiftVersion, true)
```

Comment:
For clarification, the reason we can use cache here is because this is about the availability/non availability of a given operator and this stays fairly constant throughout install, so no need to have an uncached query here?